### DISCUSSION TITLE
BindObjectAsync never returns a Promise

### DISCUSSION THREAD

**Discussion by 9xbt** [cefsharp/CefSharp#4772] [General]
I'm trying to make a settings page for my web browser, and I'm using CefSharp bindings to make the Browser save the settings to a file. My issue is that for some reason BindObjectAsync just never returns a Promise, only sometimes and it's not a lot of times. Any ideas on what i may be doing wrong?

**Comment by amaitland** [comment]
What version are you using?

What does your code look like?

> Any ideas on what i may be doing wrong?

Need more information to know what's going on.
